README.md

```diff
@@ -144,3 +147,329 @@ Otherwise you will need to tell the VM each time as well like so:
+```
+1       12200   12275   target1
+1       13505   13600   target2
+1       31000   31500   target3
+1       35138   35174   target4
+....snip....
+```
+7. PoN file (when running case samples only).  This file should be created using the Create PoN workflow (see below).
+8. Sample name (when running case samples only).  This can be extracted from the input bam file by looking at the ``SM`` tag.  Use ``samtools view -H my_sample.bam | egrep SM``.
+
+#### Case sample workflow
+
+This workflow requires a PoN file generated by the Create PoN workflow.
+
+If you do not have a PoN, please skip to the [Create PoN workflow](#create-pon-workflow), below ....
+
+##### Overview of steps
+
+- Step 1. Collect proportional coverage
+- Step 2. Create coverage profile
+- Step 3. Segment coverage profile
+- Step 4. Call segments
+
+##### Step 1. Collect proportional coverage
+
+###### Inputs
+- bam file
+- target bed file -- must be the same that was used for the PoN
+- reference_sequence (required by GATK) -- fasta file with hg19 reference.  
+
+###### Outputs
+- Proportional coverage tsv file -- MxN matrix of proportional coverage, where M is the number of targets and N is the target information and sample name.  If the file exists, it will be overwritten.
+
+```
+##fileFormat  = tsv
+##commandLine = org.broadinstitute.hellbender.tools.exome.ExomeReadCounts  ...snip...
+##title       = Read counts per target and sample
+CONTIG  START   END     NAME    SAMPLE1
+1       12200   12275   target1    1.150e-05
+1       13505   13600   target2    1.500e-05
+1       31000   31500   target3    7.000e-05
+....snip....
+```
+
+###### Invocation
+
+```
+ java -Xmx8g -jar <path_to_hellbender_jar> ExomeReadCounts -I <input_bam_file> -O <pcov_output_file_path>  -exome <target_BED> -R <ref_genome> \ 
+       -transform PCOV -exonInfo FULL -groupBy SAMPLE -keepdups
+```
+
+##### Step 2. Create coverage profile
+
+###### Inputs
+- proportional coverage file from Step 1
+- target BED file -- must be the same that was used for the PoN
+- PoN file
+- directory containing the HDF5 JNI native libraries
+
+###### Outputs
+- target file (tsv) -- details each target with chromosome, start, end, and log copy ratio estimate
+```
+#fileFormat = tsv
+#commandLine = ....snip....
+#title = ....snip....
+name    contig  start   stop    SAMPLE1
+target1    1       12200   12275   -0.5958351605220968
+target2    1       13505   13600   -0.2855054918109098
+target3    1       31000   31500   -0.11450116047248263
+....snip....
+```
+- pre-tangent-normalization target file (tsv) -- same as target file (tsv) above, but copy ratio estimates are before the noise reduction step.  The file format is the same as the target file (tsv).
+- fnt file (tsv) -- proportional coverage divided by the target factors contained in the PoN.  The file format is the same as the proportional coverage in step 1.
+- betaHats (tsv) -- used by developers and evaluators, typically, but output location must be specified.
```

Comment:
Ugh.... I left it out -- I was worried the README was already too cluttered...  Anyway, I added a brief description.

Done